deploy/helm_charts/envs/dev.yaml

```diff
@@ -44,35 +44,15 @@ ingress:
 
 serviceAccount:
   name: website-ksa
-  gsaName: website-reobot
+  gsaName: website-robot
```

Comment:
@Fructokinase is gsaName used? This typo indicates not?

Reply:
Not used, thanks for finding out.

Reply:
Removed it.